packages/http/src/validator/index.ts

```diff
@@ -141,4 +132,4 @@ function getPathParams(path: string, template: string): Dictionary<string> {
   return new URI.Template(template).match(path);
 }
 
-export { validateInput, validateOutput, validateSecurity };
+export { validateInput, validateOutput, validateSecurity, validateMediaType };
```

Comment:
Do we need that line? We usually export stuff directly in place where it is defined.

Reply:
We do not. Removed in 2c6607d